sql/2021/http/http2_3_support_by_cdn_per_page_and_request.sql

```diff
@@ -0,0 +1,28 @@
+# standardSQL
+# HTTP/2+ support per CDN by page and request
+SELECT
+  client,
+  CDN,
+  SAFE_DIVIDE(COUNTIF(http2_3 AND firstHTML), COUNTIF(firstHTML)) AS http2_3_page_pct,
+  COUNTIF(http2_3) / COUNT(0) AS http2_3_request_pct
+FROM
+  (
+    SELECT
+      client,
+      page,
+      firstHTML,
+      IFNULL(REGEXP_EXTRACT(_cdn_provider, r'^([^,]*).*'), '') AS CDN,
```

Comment:
This is only the first CDN in the list. It's ok to ignore the rest?

Reply:
Yes my understand is it should be in order, and we only the one the client connects to. I presume it's not in reverse order btw?

Reply:
I think that's correct